.gitignore

```diff
@@ -83,3 +83,6 @@ lint/generated/
 lint/outputs/
 lint/tmp/
 # lint/reports/
+
+# Mac DS files
+.DS_Store
```

Comment:
This .DS_Store file appears to still be showing up in the repo, not a big deal, but can we get rid of it?

Reply:
Yes, I'll change it!